registry.json

```diff
@@ -4207,5 +4207,19 @@
    "telegram": "xbglowx",
    "operatorName": "Brian",
    "github": "xbglowx"
+  },
+  {
+   "ticker": "NOMAD",
+   "poolId": "bbebbbf81c42de5b84fbbc82c4feab78f8bd8bcf8b5af7c73a06664a",
+   "social": {
+     "website": "https://www.nomadpool.io/",
```

Comment:
Hey I'm getting a 404 on this website, i'll approve once you confirm it's all good. Mostly doing this to make you aware

Reply:
Thanks for the heads up! I am aware of this - website is currently under construction, will be ready soon. Got the domain name ready beforehand though.